common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1609,8 +1657,8 @@ object CelebornConf extends Logging {
       .intConf
       .createWithDefault(1)
 
-  val HA_CLIENT_MAX_RETRIES: ConfigEntry[Int] =
-    buildConf("celeborn.ha.client.maxRetries")
+  val CLIENT_MAX_RETRIES: ConfigEntry[Int] =
+    buildConf("celeborn.client.maxRetries")
```

Comment:
We always use the HAClient even in non-HA mode, so I remove the "ha" to simplify the name.